docs/src/main/asciidoc/security-openid-connect-web-authentication.adoc

```diff
@@ -366,10 +371,10 @@ Note this user session can not be extended forever - the returning user with the
 
 OIDC `CodeAuthenticationMechanism` is using the default `io.quarkus.oidc.TokenStateManager' interface implementation to keep the ID, access and refresh tokens returned in the authorization code or refresh grant responses in a session cookie. It makes Quarkus OIDC endpoints completely stateless.
 
-If all of these tokens are JWT tokens then combining them may produce a session cookie value larger than 4KB and the browsers may not keep this cookie.
-In such cases, you can use `quarkus.oidc.token-state-manager.split-tokens=true` to have a unique session token per each of these three tokens.
+Note that some endpoints do not require the access token. Access token is only required if the endpoint needs to retrieve `UserInfo` or access the downstream service with this access token or use the roles in the access token (the roles in the ID token are checked by default). In such cases you can set either `quarkus.oidc.state-session-manager.stategy=id-refresh-token` (keep ID and refresh tokens only) or `quarkus.oidc.state-session-manager.stategy=id-token` (keep ID token only).
```

Comment:
```suggestion
Note that some endpoints do not require the access token.An access token is only required if the endpoint needs to retrieve `UserInfo` or access the downstream service with this access token or use the roles associated with the access token (the roles in the ID token are checked by default). In such cases you can set either `quarkus.oidc.state-session-manager.strategy=id-refresh-token` (keep ID and refresh tokens only) or `quarkus.oidc.state-session-manager.strategy=id-token` (keep ID token only).
```

Reply:
@gastaldi Sorry, I really need to be more careful, thanks for spotting these typos. Can you please edit your suggestion to have a space in `...token.An access...` ? Or I can commit and then add that space ? Whatever is simpler...

Reply:
Feel free to change it as you wish, it's your PR anyway 😉